src/lib_ccx/params.c

```diff
@@ -562,6 +562,10 @@ void print_usage (void)
 	mprint ("                       using the Chinese (Traditional) trained data\n");
 	mprint ("                       This option is also helpful when the traineddata file\n");
 	mprint ("                       has non standard names that don't follow ISO specs\n");
+	mprint ("                 -oem: Select the OEM mode for Tesseract, could be 0, 1 or 2.\n");
+	mprint ("                       0: OEM_TESSERACT_ONLY - default value, the fastest mode.\n");
+	mprint ("                       1: OEM_LSTM_ONLY - use LSTM algorithm for recognition.\n");
+	mprint ("                       2: OEM_TESSERACT_LSTM_COMBINED - both algorithms.\n");
```

Comment:
Really LSTM instead of CUBE was added only in 4.0, for lower versions we have: OEM_TESSERACT_ONLY = 0, OEM_CUBE_ONLY = 1, OEM_TESSERACT_CUBE_COMBINED = 2, and same relatively time wasting.